pkg/apis/app/v1/kieapp_types.go

```diff
@@ -127,6 +127,12 @@ type KieServerSet struct {
 	Jms                 *KieAppJmsObject `json:"jms,omitempty"`
 }
 
+type SmartRouterSet struct {
```

Comment:
Why not call it `SmartRouterObject`? Calling it Set might lead to think that it is a set of SmartRouters. For the case of `KieServerSet` it is indeed a set of KieServers that will be defined.

Reply:
Good point :)